Sources/GeneralKit/Models/TemplateSpec.swift

```diff
@@ -6,10 +6,12 @@ import Foundation
 
 public struct TemplateSpec: Codable, CustomStringConvertible {
     public let files: [File]
+    public let marked: String?
     public let suffix: String?
 
-    public init(files: [File], suffix: String? = nil) {
+    public init(files: [File], marked: String? = nil, suffix: String? = nil) {
```

Comment:
Can you explain what meaning / context for 'marked' is used here and later? I'm not a fan of this naming.